float64 pos_x
float64 pos_y
float64 pos_z
float64 linear_velocity
---
int32 result